docs/classic-ui/tinymce-customization.md

```diff
@@ -168,3 +167,46 @@ You can also exclude certain URLs from being sandboxed as follows.
 ```{seealso}
 See [`sandbox_iframes_exclusions`](https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#sandbox-iframes-exclusions) for TinyMCE's default settings.
 ```
+
+
+(classic-ui-tinymce-customization-insert-preconfigured-html-blocks-label)=
+
+## Insert preconfigured HTML blocks
+
+You can add custom HTML blocks in TinyMCE.
+You can insert them in your content using the TinyMCE {menuselection}`Insert Template` menu option using the [TinyMCE Template plugin](https://www.tiny.cloud/docs/tinymce/6/template/).
+
+```{important}
+The Template plugin has been completely removed from the TinyMCE 7.0 release used in Plone 6.1.
+As an alternative solution, you can use the [Advanced Template Premium plugin](https://www.tiny.cloud/docs/tinymce/6/advanced-templates/). 
+```
+
```

Comment:
This can be removed since we've backported the community edition of `template` plugin to TinyMCE 7

Reply:
Done in https://github.com/plone/documentation/pull/1870/commits/2d47570f0de137adfdaee38dbfed18bd47ed1888. Thanks @petschki! Will merge the chain of PRs when all green.